cli/ts/commands/genProofs.ts

```diff
@@ -412,4 +418,6 @@ export const genProofs = async (
   const tallyEndTime = Date.now();
 
   logYellow(quiet, info(`gen tally proof took ${(tallyEndTime - tallyStartTime) / 1000} seconds\n`));
+
+  return tallyFileData;
```

Comment:
thoughts: maybe for another PR but if `genProof()` function have to return data, it should return proof, rather tally data

Reply:
it will - https://github.com/privacy-scaling-explorations/maci/pull/832 it's in there, so leaving it out of this one for now